rewrite-maven/src/main/java/org/openrewrite/maven/AddDependencyVisitor.java

```diff
@@ -16,9 +16,12 @@
 package org.openrewrite.maven;
 
 import lombok.RequiredArgsConstructor;
+import org.jetbrains.annotations.NotNull;
```

Comment:
We try to avoid this annotation. Use JSpecify on the package instead where possible.

Reply:
W00ps good call! IntelliJ added at when refactoring 😢.